Pass block-workflow parameter to until_front_of_line step [semver:patch]

mu-bro opened the pull request:
### Motivation, issues

**_block-workflow_** parameter is not passed from **queue/block_workflow** job to **until_front_of_line** step, therefore block-workflow is always set to 'false'.

mu-bro committed:
Add debug

Commit:
Pass block-workflow parameter to until_front_of_line step

Commit:
remove debug

eddiewebb commented:
Can you merge in latest from master?

eddiewebb commented:
bors try

eddiewebb committed:
Merge branch 'master' into master

eddiewebb commented:
bors try

eddiewebb commented:
BotComment: *Development* version of orb available for manual validation - `eddiewebb/queue@dev:50`

eddiewebb committed:
add test for job passing workflow param

This important pass was missed, this test ensures our default job's logic properly assumes it should block workflow.

eddiewebb commented:
I added some tests (bats is a little cryptic, i dont expect folks to take that ramp), assuming these run through it LGTM!

Thanks for catching this.

eddiewebb commented:
bors merge

eddiewebb commented:
BotComment: *Development* version of orb available for manual validation - `eddiewebb/queue@dev:50`

eddiewebb commented:
BotComment: *Production* version of orb available for use - `eddiewebb/queue@1.4.1`
